runtime/autoload/health/provider.vim

```diff
@@ -502,10 +502,10 @@ function! s:check_node() abort
     return
   endif
 
-  if !executable('node') || !executable('npm')
+  if !executable('node') || !executable('npm') || !executable('yarn')
```

Comment:
with this condition, checkhealth will warn if yarn isn't installed, even if npm is. how about:

    if !executable('node') || (!executable('npm') && !executable('yarn'))